WordPress/Sniffs/Theme/NoHardCodedUrlsSniff.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * WordPress_Sniffs_Theme_NoHardCodedUrlsSniff File Doc Comment
+ *
+ * @category WordPress_Sniffs_Theme_NoHardCodedUrlsSniff
+ * @package  PHP_Codesniffer
+ * @author   khacoder
+ * @license  http://www.gnu.org/copyleft/gpl.html GNU General Public License
+ * @link     http://kevinsspace.ca
+ */
```

Comment:
It'd be great if you could follow the example from:
https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress/Sniffs/VIP/AdminBarRemovalSniff.php#L1-L16